### PR TITLE
dts: msm8953: Add support for Motorola Moto G6

### PR DESCRIPTION
**Pull request by FieryFlames**
Adds a device tree for the Motorola Moto G6.
The bootloader requires every appended device tree has the `model` property, otherwise it will crash trying to select a device tree. 
We could add proper model props to every device tree under msm8953, or just add an empty string model to `skeleton64.dtsi`?
`msm8952`'s `cedric` is also affected by this quirk.

### CODE REVIEW

**Comment by a-andre** [comment]
Thanks @FieryFlames. I tried your `model` suggestion with motorola-cedric.
It improve the situation, but there is still a problem with `$(LOCAL_DIR)/msm8937-mtp.dtb`. I have to remove that from `ADTBS`  of `rules.mk` to detect cedric, otherwise lk2nd shows `Failed to find matching lk2nd device node: -1`

**Comment by FieryFlames** [comment]
> Thanks @FieryFlames. I tried your `model` suggestion with motorola-cedric. It improve the situation, but there is still a problem with `$(LOCAL_DIR)/msm8937-mtp.dtb`. I have to remove that from `ADTBS` of `rules.mk` to detect cedric, otherwise lk2nd shows `Failed to find matching lk2nd device node: -1`

I had someone test a build on cedric for me and it worked.
Did you add `model = "cedric";` to Cedric's dtb like my Ali dtb? moto's selection logic uses this.
You might also need to remove cedric from qcdtbs.

File: lk2nd/device/dts/msm8953/sdm450-motorola-ali.dts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-2.0-only
```
Comment (TravMurav) [inline lk2nd/device/dts/msm8953/sdm450-motorola-ali.dts:1]:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```

I'd prefer bsd3 for new dt submissions, like what linux prefers nowdays

**Comment by TravMurav** [comment]
Could you please squash last two commits into first? So one that adds the device, and one that adds the quirk to lk2nd.dtsi + build system. Otherwise it's good to go I think

**Comment by FieryFlames** [comment]
> Could you please squash last two commits into first? So one that adds the device, and one that adds the quirk to lk2nd.dtsi + build system. Otherwise it's good to go I think

Done 🙂

**Comment by TravMurav** [comment]
Thanks!

**Comment by a-andre** [comment]
> > Thanks @FieryFlames. I tried your `model` suggestion with motorola-cedric. It improve the situation, but there is still a problem with `$(LOCAL_DIR)/msm8937-mtp.dtb`. I have to remove that from `ADTBS` of `rules.mk` to detect cedric, otherwise lk2nd shows `Failed to find matching lk2nd device node: -1`
> 
> I had someone test a build on cedric for me and it worked. Did you add `model = "cedric";` to Cedric's dtb like my Ali dtb? moto's selection logic uses this. You might also need to remove cedric from qcdtbs.

With the model property added, moto's selection prefers some other board-id (`QCOM_BOARD_ID_MTP`). Adding `model = "cedric"` didn't help. See #416 for a solution.

**Pull request merged**